### ISSUE TITLE
Would you release your source code of your paper?

### ISSUE THREAD

**Issue by laulampaul** [thnkim/thnkim.github.com#1]

**Comment by ChristopherDrum** [comment]
I second this. I do a lot of personal art projects with found imagery from 70's and 80's magazines. Having an excellent descreening methodology would really be great. I don't even need the training data; I have extensive experience in graphic arts and can generate my own dataset if need be.

**Comment by ChristopherDrum** [comment]
@laulampaul I have had good success by using a combination of other techniques to descreen.

1. Scale the image down by 33% (in Photoshop, Image Size > Resample > Bicubic (preserve gradients worked well for me)
2. Using Waifu2x, upscale by 1.5x with noise removal
3. Do a little cleanup in photo editing software.

The Waifu2x super-resolution upscaler introduces a strange texture sometimes, but the results are still much better than anything I could reproduce by hand.
